bindings/wasm/examples/browser/resolve_history.js

```diff
@@ -0,0 +1,201 @@
+import {getExplorerUrl, logExplorerUrlToScreen, logObjectToScreen, logToScreen} from "./utils.js";
+import {createIdentity} from "./create_did.js";
+
+import * as identity from "../../web/identity_wasm.js";
+
+/**
+ Advanced example that performs multiple diff chain and integration chain updates and
+ demonstrates how to resolve the DID Document history to view these chains.
+
+ @param {{defaultNodeURL: string, explorerURL: string, network: Network}} clientConfig
+ @param {boolean} log log the events to the output window
+ **/
+export async function resolveHistory(clientConfig, log = true) {
+    if (log) logToScreen("Resolve History Example");
+
+    // Create a default client configuration from network.
+    const config = identity.Config.fromNetwork(clientConfig.network);
+
+    // Create a client instance to publish messages to the Tangle.
+    const client = identity.Client.fromConfig(config);
+
+    // ===========================================================================
+    // DID Creation
+    // ===========================================================================
+
+    // Create a new identity (see "create_did.js" example).
+    const {doc, key, receipt: originalReceipt} = await createIdentity(clientConfig, false);
+
+    // ===========================================================================
+    // Integration Chain Spam
+    // ===========================================================================
+
+    // Publish several spam messages to the same index as the integration chain on the Tangle.
+    // These are not valid DID documents and are simply to demonstrate that invalid messages can be
+    // included in the history, potentially for debugging invalid DID documents.
+    const intIndex = doc.integrationAddress();
```

Comment:
Unfortuanetly the term "address" is incorrect. Only IOTA tokens use an address, otherwise it is called an index.

Reply:
Thanks for the clarification, this affects `diff_address -> diff_index` too.